sandbox/libs/authn/src/main/java/org/opensearch/authn/AuthenticationTokenHandler.java

```diff
@@ -28,12 +31,16 @@ public class AuthenticationTokenHandler {
      * @param authenticationToken the token from which to extract
      * @return the extracted shiro auth token to be used to perform login
      */
-    public static AuthenticationToken extractShiroAuthToken(org.opensearch.authn.tokens.AuthenticationToken authenticationToken) {
+    public static AuthenticationToken extractShiroAuthToken(org.opensearch.authn.tokens.AuthenticationToken authenticationToken)
+        throws RuntimeException, BadCredentialsException {
```

Comment:
Does this method throw either of these exceptions?

Reply:
It throws badCredentials--I removed the old runtime exception.

Reply:
It doesn't throw the BadCredentialsException either, I see its in the signature of `handleBearerAuth` but it can be removed from there too.